public/views/answers.html

```diff
@@ -1,89 +1,69 @@
+      <h4 class="center-align">What would you like to do now?</h4>
       You might consider donating to charity to atone for the horrifying answers you've submitted. Donating will also place a lovely crown on your avatar's head.
-    </div>
-    <div class="game-end-answer-text" ng-show="game.state === 'game ended' || game.gameWinner !== -1">
-      <h2>What would you like to do now?</h2>
+    </p>
+    <p class="center-align" ng-show="game.state === 'game ended' || game.gameWinner !== -1">
+      <h4 class="center-align">What would you like to do now?</h4>
       Help yourself to another game or help a child in need? Donating will also place a lovely crown on your avatar's head.
+    </p>
+    <div class="col s12">
+      <a href='/#!/app' class="col offset-l3 l2 btn btn-large app-buttons" ng-click="game.joinGame('joinNewGame')">Join a New Game</a>
+      <a href='/' class="btn btn-large col l2 offset-l2">Return to Lobby</a>
     </div>
-    <div id='inner-text-container'>
-      <a href='/#!/app'>
-      <div id="join-new-game" ng-click="game.joinGame('joinNewGame')">
-        Join a New Game
-      </div></a>
-      <a href='/'>
-        <div id="exit-match">
-        Return to Lobby
-        </div>
-      </a>
-    </div>
-  </div>
-
-  <div id="charity-widget-container">
-      <!-- This is real -->
+    <div>
       <a href="http://www.crowdrise.com" id="crowdriseStaticLink-fundraiser-121142" title="Fundraising Websites on Crowdrise">Fundraising Websites - Crowdrise</a>
       <script type="text/javascript" src="https://www.crowdrise.com/widgets/donate/fundraiser/121142/?callback=userDonationCb"></script>
-      <!-- This is for testing -->
-      <!-- <a href="http://www.crowdrise.com" id="crowdriseStaticLink-project-36715" title="Fundraising Websites on Crowdrise">Fundraising Websites - Crowdrise</a> -->
-      <!-- <script type="text/javascript" src="https://apisandbox.crowdrise.com/widgets/donate/project/36715/?callback=userDonationCb"></script> -->
+    </div>
   </div>
-</div>
-<div id="czar-blank-container" ng-show="isCzar() && game.table.length===0 && game.state !== 'game dissolved' && game.state !== 'awaiting players'">
-  <div id="czar-blank-inner">
-    You are the Card Czar. <br>
-    <span id="smaller-text">Players are choosing their answers. Prepare to select a winner.</span>
+  <div ng-show="isCzar() && game.table.length===0 && game.state !== 'game dissolved' && game.state !== 'awaiting players'">
+    <h4 class="center-align">You are the Card Czar</h4>
+    <p>Players are choosing their answers. Prepare to select a winner.</p>
+    <h4 class="center-align">Did you know?</h4>
+    <p>{{ makeAWishFact }}</p>
   </div>
-  <div id='charity-fact-container'>
-    <div id='charity-fact-tagline'>Did you know?</div>
+  
 
-    <div id='charity-fact'>
-       {{ makeAWishFact }}
-    </div>
+  <div ng-cloak ng-hide="game.state === 'game ended' || game.state ==='game dissolved' || game.state === 'awaiting players'">
```

Comment:
You can wrap the condition in a function attached to a scope to make it more readable.